site/_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<footer role="contentinfo">
+<footer>
```

Comment:
You added these in https://github.com/jekyll/jekyll/pull/3256 – should we modify the validation to accept them, or are they no longer needed?